org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/BuilderStateDiscarder.java

```diff
@@ -34,12 +37,34 @@
 public class BuilderStateDiscarder {
 
 	private static final Logger logger = Logger.getLogger(BuilderStateDiscarder.class);
+	private static final MethodHandle InternalBuilder_getLastBuiltTree;
 
+	static {
+		try {
+			@SuppressWarnings("restriction")
+			MethodHandles.Lookup lookup = MethodHandles.privateLookupIn(
```

Comment:
i assume if we keep java 8 we need to do similar old school reflection as in JavaBuilderState / BuildManagerAccess